gulp/config.js

```diff
@@ -0,0 +1,51 @@
+var dest = './doc/public',
+    src = './pattern-library',
+    siteDir = './_site',
+    docDir = './doc';
+
+module.exports = {
+    documentation: {
+        testing: {
+            sources: [
+                './src/js/utils/spec-helpers/*.js'
+            ],
+            output: './doc/_testing',
+            viewClass: 'testing'
+        },
+        utilities: {
+            sources: [
+                './src/js/utils/*.js'
+            ],
+            output: './doc/_utilities',
+            viewClass: 'utility'
+        },
+        views: {
+            sources: [
+                './src/js/disclosure/*.js',
+                './src/js/dropdown-menu/*.js',
+                './src/js/pagination/*.js'
```

Comment:
Can we use a path like `'./src/js/**/*.js'` here to avoid having to remember to add new patterns here when we create them?